### DISCUSSION TITLE
Should we restore Windows support? How?

### DISCUSSION THREAD

**Discussion by conradoplg** [ZcashFoundation/zebra#3704] [Q&A] [answered]
Recent updates to `zcashd` code, which is used by the  `zcash_script` crate, break when compiling for Windows with MSVC compiler (the GitHub Actions default).

But we can't use MingW since `rocksdb` does not support it.

In https://github.com/ZcashFoundation/zebra/pull/3668 the Windows support was disabled so that we don't block on this, considering that Windows support is not critical at this time.

But what should we do next?

- Keep not supporting Windows
- Make `zcashd` compile with MSVC (may be a simple issue, but note that `zcashd` does not officially support Windows)
- Make `rocksdb` compile with MingW (see this [draft PR](https://github.com/rust-rocksdb/rust-rocksdb/issues/469))
- Something else?

**Comment by teor2345** [comment]
> * Something else?

Delay this decision until we have users who want to use Windows?

**Comment by conradoplg** [comment, marked as answer]
In yesterday's meeting we decided to not support Windows for now and create these tickets:

- https://github.com/ZcashFoundation/zebra/issues/3800
- https://github.com/ZcashFoundation/zebra/issues/3801
